config/overlays/tilt-local-dev/operator-controller/patches/dev-deployment.yaml

```diff
@@ -8,6 +8,3 @@
 - op: remove
   # remove --leader-elect so container doesn't restart during breakpoints
   path: /spec/template/spec/containers/0/args/2
-- op: add
-  path: /spec/template/spec/containers/0/args/-
-  value: --feature-gates=PreflightPermissions=true
```

Comment:
Should it be really removed?

Reply:
Yes, Joe asked for it to be removed.
https://github.com/operator-framework/operator-controller/pull/1790#discussion_r1960609934